### PR TITLE
Fix input field height

### LINKED ISSUE #2149: Fix height of form input fields
Currently, the elements below the input-fields in forms move downward when warning messages are inserted.

- the expected behaviour
The elements should not move from their own position.

- the observed behaviour
The elements move downward from their position.
screenshot_2019-02-23 evalai

### PR DESCRIPTION
**Pull request by geekayush**
Fix #2149 : Make the height of form input fields constant

### CODE REVIEW

File: frontend/src/css/modules/auth.scss [hunk 1]
```diff
@@ -50,6 +49,13 @@ img.auth-logo{
     min-height: 100vh;
     overflow: hidden;
     position: relative;
+    input{
```
Comment (Ram81) [inline frontend/src/css/modules/auth.scss]: @geekayush please add a space before `{` and newline above this line.

File: frontend/src/css/modules/auth.scss [hunk 2]
```diff
@@ -50,6 +49,13 @@ img.auth-logo{
...
+    margin-bottom: 0px;
+    }
+    .input-field{
```
Comment (Ram81) [inline frontend/src/css/modules/auth.scss]: Same here

File: frontend/src/css/modules/auth.scss [hunk 3]
```diff
@@ -50,6 +49,13 @@ img.auth-logo{
...
+    height: 72px;
+    max-height: 72px;
```
Comment (Ram81) [inline frontend/src/css/modules/auth.scss]: Can you also indent it properly

**Comment by RishabhJain2018** [comment]
@geekayush Any update on this? Can you please also add a gif of the UI.

**Comment by geekayush** [comment]
> @geekayush Any update on this? Can you please also add a gif of the UI.

Sure

**Comment by geekayush** [comment]
@RishabhJain2018 
### Before:

### After:

**Comment by RishabhJain2018** [comment]
Thanks for the fix @geekayush.

**Pull request merged**